web-ng/src/app/services/router/router.service.ts

```diff
@@ -169,11 +169,35 @@ export class NavigationService {
   }
 
   /**
-   * Navigate to the URL with new project id
+   * Navigate to the URL with new project id.
```

Comment:
Can we rename this to `selectProject` while we're at it, since it doesn't just set the id, I also removes other params in the process.